src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb

```diff
@@ -106,6 +106,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' </summary>
         Friend MustOverride Function GetCrefReferenceSymbolInfo(crefReference As CrefReferenceSyntax, options As SymbolInfoOptions, Optional cancellationToken As CancellationToken = Nothing) As SymbolInfo
 
+        Public Overrides Function WithSuppressAccessChecks() As SemanticModel
+            Dim moduleSymbol = TryCast(Compilation.SourceModule, SourceModuleSymbol)
```

Comment:
If the current model is already ignoring accessibility, we should just return it.